.github/workflows/ci.yml

```
@@ -121,7 +121,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9"]
+        python-version: ["3.8", "3.9", "3.10"]
```

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10"]
```
I think that, since we are supporting above Python 3.7, shouldn't it be included in this matrix testing?

---

It is tested in job Build_and_Test. We separated them to be able to trigger them independantly. The CI will however be reworked soon with a testing strategy coherent with what is being put in place for PyDPF-Core.